pkgs/applications/editors/neovim/tests.nix

```diff
@@ -132,4 +139,6 @@ rec {
     extraName = "-pathogen";
     configure.pathogen.pluginNames = [ "vim-nix" ];
   };
+
+  test_nvim_with_lua_packages = wrapNeovim2 "-with-lua-packages" nvimWithLuaPackages;
```

Comment:
the test should start neovim to check if the require works.

Reply:
Done